.github/workflows/balena-ci.yml

```diff
@@ -16,9 +16,15 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: balena-io/balena-ci@master
+        name: Deploy to balena-cloud.com
+        with:
+          balena_token: ${{ secrets.BALENA_TOKEN }}
+          fleet: 'balena_os/${{ matrix.arch }}-supervisor'
+          versionbot: true
+      - uses: balena-io/balena-ci@master
```

Comment:
How about pushing on prod & staging in parallel?
```yml
jobs:
  deploy-release:
    runs-on: ubuntu-latest
    strategy:
      fail-fast: true
      matrix:
        arch: [aarch64, amd64, armv7hf, i386, rpi]
        environment: [ { tld: balena-cloud.com, token_prop_name: BALENA_TOKEN }, { tld: balena-staging.com, token_prop_name: BALENA_STAGING_TOKEN } ]
    steps:
      - uses: actions/checkout@v2
      - uses: balena-io/balena-ci@master
        name: 'Deploy to ${{ matrix.environment.tld }}'
        with:
          balena_token: ${{ secrets[matrix.environment.token_prop_name] }}
          fleet: 'balena_os/${{ matrix.arch }}-supervisor'
          versionbot: true
          environment: ${{ matrix.environment.tld }}

```

Reply:
I like it! added